tests/src/system/basic/Swift3WhiskObjectTests.scala

```diff
@@ -64,30 +68,31 @@ class Swift3WhiskObjectTests
             }
 
             // create an action that fires the trigger
-            val file = TestUtils.getTestActionFilename("triggerEvent.swift")
+            val file = TestUtils.getCatalogFilename("/samples/trigger.swift")
             val actionName = "ActionThatTriggers"
             assetHelper.withCleaner(wsk.action, actionName) { (action, _) =>
                 action.create(name = actionName, artifact = Some(file), kind = Some("swift:3"), parameters = Map("key0" -> "value0".toJson), expectedExitCode =0)
             }
 
             // invoke the action
-            val invokeResult = wsk.action.invoke(actionName, Map("key0" -> "value0".toJson), blocking = true, result = false, 0).stdout
+            val run = wsk.action.invoke(actionName, Map("key0" -> "value0".toJson))
+            withActivation(wsk.activation, run) {
+                activation =>
```

Comment:
you'll be Scala-fu fighting in no time.

`withActivation` waits for the activation to complete - since you're polling for the activation list to hit a specific number (2), and not using the `activation`, omit both lines (and the matching curly).